apps/pie-docs/src/foundations/typography/code.md

```diff
@@ -103,9 +104,9 @@ The basic strategy is to load in the `JETSansDigital-Regular` webfont as soon as
    })();
    ```
 
-   This lazily loads in the `JETSansDigital-Bold` and `JETSansDigital-ExtraBold` fonts to take over from the Faux Bold rendering of the Regular font.
+  This lazily loads in the `JETSansDigital-Bold` and `JETSansDigital-ExtraBold` fonts to take over from the faux bold rendering of the regular font.
 
-   N.b. You should also put the two webfont URLs into your service worker so that they are cached at browser level, saving additional requests on repeat vists.
+  N.b. You should also put the two webfont URLs into your service worker so that they are cached at browser level, saving additional requests on repeat visits.
```

Comment:
Doesn't have to be done now, but should we use an info notification here instead?

Reply:
Do this in a later PR, not needed for now :) (I'm not Jordi but at this point lets not making changes like this)

Reply:
Echo that, let's add this to a future pr please